modules/fundamental/Cargo.toml

```diff
@@ -78,6 +78,14 @@ urlencoding = { workspace = true }
 walkdir = { workspace = true }
 zip = { workspace = true }
 
+opentelemetry = { workspace = true }
+tracing-opentelemetry = { workspace = true }
+opentelemetry_sdk = { workspace = true }
+opentelemetry-otlp = { workspace = true, features = ["grpc-tonic"] }
+tracing = { workspace = true, features = ["std"]}
+tracing-core = "0.1" # We are not using this in other OTEL-related code, don't need to add in root Cargo.toml for now.
```

Comment:
Everything should go to the workspace.

Reply:
sure thing thanks for the review I'll also change the original criterion bench :+1: